pages/detail-product/detail-content.pug

```diff
@@ -0,0 +1,131 @@
+mixin detail-content
+    .wrapper-detail
+            .container-sm.product__detail-container
+                .row.product__detail
+                    .product__detail-img.col-md-5
+                        .img-parent
+                            +img('assets/detail.png')
+                        .img-child 
+                            +img('assets/detail2.png')
+                            +img('assets/detail3.png')
+                            +img('assets/detail4.png')
+                            +img('assets/detail5.png')
+                            +img('assets/detail6.png')
+                    .product__detail-desc.col-md-7
+                        .product__detail-name
+                            span Cây Dạ Lam
+                        .product__detail-rate
+                            i.ri-star-fill
+                            i.ri-star-fill
+                            i.ri-star-fill
+                            i.ri-star-fill
+                            i.ri-star-fill
```

Comment:
Tạo 1 mixin riêng cho rating. Với params truyền vào là số sao
Giả sử rating(4) thì in ra 4 sao, rating(5) thì in 5 sao
Ngoài ra có thể làm 3.5 sao, 4.5 sao nếu có thể
Đồng thời tạo cho nó 1 component riêng. Chứ k để tên class là product__detail-rate. Vì ngoài product detail ra, product card cũng có thể sử dụng style, mixin này